test/docs/OpenJ9TestUserGuide.md

```diff
@@ -0,0 +1,302 @@
+
+```
+cd openj9/test/TestConfig
+export JAVA_BIN=<path to JDK bin directory that you wish to test>
+export SPEC=linux_x86-64_cmprssptrs
+make -f run_configure.mk
+make test
+```
+
+## Prerequisites:
+Please read [Prerequisites.md](./Prerequisites.md) for details.
+
+# Tasks in OpenJ9 Test
+
+1. Configure environment:
+
+  * required environment variables
+    ```
+    JAVA_BIN=<path to JDK bin directory that you wish to test>
+    SPEC=[linux_x86-64|linux_x86-64_cmprssptrs|...] platform_on_which_to_test
+    JAVA_VERSION=[SE80|SE90|SE100|Panama|Valhalla] (SE90 default value)
+    JAVA_IMPL=[openj9|hotspot|sap] (openj9 default value)
+    ```
+
+  * dependent libs for tests
+
+Please read [DependentLibs.md](./DependentLibs.md) for details.
+
+2. Compile tests:
+
+  * compile and run all tests
+    ```
+    make test
+    ```
+
+  * only compile but do not run tests
+    ```
+    export BUILD_LIST=comma_separated_projects_to_be_compiled (defaults to all projects)
+    make compile
+    ```
+
+3. Add more tests:
+
+  * for Java8/Java9 functionality
+
+    - If you have added new features to OpenJ9, you will likely
+    need to add new tests. Check out openj9/test/TestExample for
+    the format to use.
+
+    - If you have many new test cases to add, then you may want to
+    copy the TestExample project, change the name,replace the test
+    examples with your own code, update the build.xml and playlist.xml
+    files to match your new Test class names. The playlist.xml format 
+    is defined in TestConfig/playlist.xsd.
+
+    - A test can be tagged with following elements:
+      - level:   [sanity|extended] (extended default value)
+      - group:   [functional|openjdk|external|perf|jck|system] (required 
+                 to provide one group per test)
+      - impl:    [openj9|hotspot] (filter test based on exported JAVA_IMPL 
+                 value; a test can be tagged with multiple impls at the 
+                 same time; default to all impls)
+      - subset:  [SE80|SE90|SE100|Panama|Valhalla] (filter test based on 
+                 exported JAVA_VERSION value; a test can be tagged with 
+                 multiple subsets at the same time; default to all subsets)
+
+    - Most OpenJ9 FV tests are written with TestNG. We leverage
+    TestNG groups to create test make targets. This means that
+    minimally your test source code should belong to either
+    level.sanity or level.extended group to be included in main
+    OpenJ9 builds.
+
+4. Run tests:
+
+  * group of tests <br />
+    make _group <br />
+    e.g., 
+    ```
+    make _functional
+    ```
+
+  * level of tests <br />
+    make _level <br />
+    e.g., 
+    ```
+    make _sanity
+    ```
+
+  * level of tests with specified group <br />
+    make _level.group <br />
+    e.g., 
+    ```
+    make _sanity.functional
+    ```
+
+  * a specific individual test <br />
+    make _testTargetName <br />
```

Comment:
May want to explain difference between _testTargetName and _testTargetName_0, etc (the suffix number refers to the variation in the playlist.xml file).